pynetgear/__init__.py

```diff
@@ -143,6 +154,10 @@ def login_v2(self):
             _LOGGER.debug(response.headers)
             return False
 
+        # check login succes with info call
```

Comment:
Don't you think the problem is v1 specific and we could avoid this for v2 since we have a cookie at this point of the code ?
This is an extra call, even if it's not a biggie either since `login` shouldn't be called that often.

I have no material to test and never experienced the problem, and it could be worth to do it anyway as a safety, not sure. I'll let you judge on that :)

Reply:
Yea I was wondering on this myself, I know it is a problem with v1 login, not sure if this could also happen at v2, but included it as an extra safety.
I will remove it and if we get any reports from users having issues I will add it again to v2.